build.gradle

```diff
@@ -266,6 +211,32 @@ task cleanBundles(type: Delete) {
 
 tasks.clean.finalizedBy(cleanBundles)
 
+task helmLint(type: Exec) {
```

Comment:
If you can, can you have these run after the distributions task?
The distributions should create the deployment directory under the build directory.  You could then point the validations to the directory:
` project.buildDir/deployment/helm/synopsys-alert`
Could be something to look into after this release.